services/vision/builtin/tflite_classifier.go

```diff
@@ -82,9 +83,9 @@ func NewTFLiteClassifier(ctx context.Context, conf *vision.VisModelConfig,
 		if err != nil {
 			return nil, err
 		}
-
 		classifications, err := unpackClassificationTensor(ctx, outTensor, model, labels)
 		if err != nil {
+			logger.Error(err)
```

Comment:
What is the best way to surface this error to the user? Logging it here logs it over and over and over again, but just returning the error doesn't seem to surface it

Reply:
Yeah this function that gets returned will run every time an image gets classified. My first thought is that if we hate that, maybe try a logger.Fatal() which will kill everything after printing it out once. If that seems like overkill, maybe only fatal depending on the type of error. Either way it seems like don't want to keep on trying to classify this over and over again if we're definitely not unpacking the tensors correctly.

Reply:
hmm the linter complains about `logger.Fatal`, do you know of a workaround?